table/merging_iterator.cc

```diff
@@ -798,7 +800,6 @@ void MergingIterator::SeekImpl(const Slice& target, size_t starting_level,
     active_.erase(active_.lower_bound(starting_level), active_.end());
   }
 
-  status_ = Status::OK();
```

Comment:
Some child iterators' status are checked above in `AddToMinHeapOrCheckStatus(&children_[level]);`, should not overwrite here.